.github/workflows/golangci-lint.yml

```diff
@@ -20,7 +20,8 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
```

Comment:
This might need to be v4

Reply:
Ah just updated, it needs to be minimum v3!